tests/gold_tests/custom_routing/parent_basic.test.py

```diff
@@ -0,0 +1,101 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pop
```

Comment:
This import is failing on the CI machine.  Causes an exception.

Reply:
I can install pop on the CI machines to solve that issue easily enough. I'm concerned that the test/Pipfile isn't updated with this new Python package requirement. That is, I can fix things for CI by installing pop on the CI machines that run autest, but for our local autest runs pop will need to be installed which will require the Pipfile update.

I'll hold off on udpating the CI machines until the Pipfile in the PR is updated with this and any other requirements.